lib/BridgedRoom.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+function BridgedRoom(bridge, gitter, gitterRealtime, matrixRoomModel, gitterRoomModel) {
```

Comment:
Addendum: Please also document the args via [JSDoc3](http://usejsdoc.org/about-getting-started.html). This becomes important for the reader as I do not know if it is valid to pass an `undefined` value for `matrixRoomModel` for example. If it **is** valid, then `:12` indicates a null-pointer exception and should be null guarded. If it is **not** valid, then I would question whether you want to fail fast and do a null guard check here and throw if it is `undefined` or `null`.